File: JAVA/Exercicios/src/exercicios_java/Ex05.java
package exercicios_java;

import java.util.Scanner;

public class Ex05 {

	public static void main(String[] args) {
		// Entrar via teclado com o valor de uma temperatura em graus Celsius, calcular
		// e exibir sua temperatura equivalente em Fahrenheit.

		Scanner ler = new Scanner (System.in);
		
		double c, f;
		
		System.out.printf("Digite a temperatura em Celsius: ");
		c = ler.nextDouble();
		
		f = (c * 9/5) + 32;
		
		System.out.printf("A temperatura convertida para F é %.2f", f);
		
		
		
		
	}

}

File: Python/Exercicios/ex05.py
#Entrar via teclado com o valor de uma temperatura em graus Celsius, calcular e exibir sua temperatura equivalente em Fahrenheit.
c = float(input('Digite o valor em Celsius na sua cidade: '))
f = (1.8*c) + 32 
print(f'O valor da temperatura em Fahrenheit será de: {f:.2f}')